flight_planning/v1/flight_planning.yaml

```diff
@@ -147,11 +153,46 @@ components:
         intended_flight:
           $ref: '#/components/schemas/FlightPlan'
         request_id:
+          type: string
           description: >-
             ID uniquely identifying the upsertion request.  If additional requests are received with the same
             request_id, the response from the first request should be returned, or an error indicated.
-          anyOf:
-            - $ref: './astm_f3548_21.yaml#/components/schemas/UUIDv4Format'
+
+    PlanningResult:
+      type: string
+      description: >-
+        The result of a flight planning activity.
+
+          - `Planned`: The data submitted in the flight plan intent was valid and the flight plan was successfully processed by the USS and is now authorized, but the user may not yet start flying (even if within the time bounds of the flight plan).
+
+          - `ReadyToFly`: The flight plan is ready for the operator to nominally fly within the bounds (including time) of the flight plan.
+
+          - `OffNominal`: The flight plan now reflects the operator's actions, but the flight plan is not in a nominal state (e.g., the USS has placed the ASTM F3548-21 operational intent into one of the Nonconforming or Contingent states).
+
+          - `Rejected`: The data provided in the flight plan was invalid and/or could not be used to successfully authorize or update the flight.  The reasons for rejection may include a disallowed conflict with another flight.
+        
+          - `Closed`: The flight plan was closed successfully by the USS and is now out of the UTM system.
+
+          - `Failed`: The USS was not able to successfully authorize or update the flight plan due to a problem with the USS or a downstream system.
+
+          - `NotSupported`: The USS's implementation does not support the attempted interaction.  For instance, if the request specified a high-priority flight and the USS does not support management of high-priority flights.
+      enum: [Planned, ReadyToFly, OffNominal, Rejected, Failed, NotSupported]
+      example: Planned
+
+    AdvisoryInclusion:
+      type: string
+      description: >-
+        Indication of whether any advisories or conditions were provided to the user along with the result of an
+        associated flight planning attempt.
+
+          - `Unknown`: It is unknown or irrelevant whether advisories or conditions were provided to the user
+
+          - `AtLeastOneAdvisoryOrCondition`: At least one advisory or condition was provided to the user.
+
+          - `NoAdvisoriesOrConditions`: No advisories or conditions were provided to the user.
+      enum: [Unknown, AtLeastOneAdvisoryOrCondition, NoAdvisoriesOrConditions]
+      default: Unknown
+      example: No
```

Comment:
This "no" isn't in the enum list. Perhaps "NoAdvisoriesOrConditions"?

Reply:
Thanks for the catch -- updated.